docs/src/architecture/08_concepts/signed_doc/cddl/collaborators.cddl

```diff
@@ -0,0 +1,8 @@
+; collaborators
+
+
+; Allowed Collaborators on the next subsequent version of a document.
+collaborators = [ * catalyst_id_kid ]
+
+; UTF8 Catalyst ID URI encoded as a bytes string.
+catalyst_id_kid = bytes
```

Comment:
What do you think to change it into this ?
So we will need to encode a CBOR catalyst id string as CBOR bytes ?
```suggestion
catalyst_id_kid = bytes .cbor text
```

Reply:
I need to double check but as far as I know this is not the same as storing utf8 in the bytes.  What it is saying is. These bytes are cbor encoded and when you decode it use this type.  So it's wrapping the utf8 inside a cbor text field, which is then wrapped inside a bytes field.  It's just overhead with no real gain.

Equivalent to in Jason:
{
  "bytes" : "{ \"text\": \"catalyst id\" } "
}
Instead of just:
{
  "bytes" : "catalyst id"
}

Reply:
Utf8 is a legitimate byte stream.

Reply:
The ideal would be let kid be "bytes / text" but that won't be Close standard anymore and normal close libraries would fail decoding it.

Reply:
Ok, yes, with this suggestion, this:

```
4B                        # bytes(11)
   636174616C797374206964 # "catalyst id"
```

Becomes this:
```
4C                          # bytes(12)
   6B636174616C797374206964 # "kcatalyst id"
```
where the bytes value further decodes as:
```
6B                        # text(11)
   636174616C797374206964 # "catalyst id"
```
I think that's likely to cause specification interpretation errors, and doesn't meaningfully improve the process of decoding the catalyst id.